ordinals/hooks/pre-start

```diff
@@ -37,7 +37,48 @@ if [[ ! -f "${APP_DATA_DIR}/AT_LEAST_0-18-1" ]]; then
     touch "${APP_DATA_DIR}/AT_LEAST_0-18-1"
 fi
 
+# migrate all wallet database files to ord/wallets, ord/testnet3/wallets, ord/regtest/wallets, and ord/signet/wallets for pre-0.18.2 versions: https://github.com/ordinals/ord/releases/tag/0.18.2
```

Comment:
Hmm, I'll have to double check but I didn't think ord stored the wallets in redb files.  I thought when you do wallet create it creates a folder with a wallet.dat file in the bitcoin directory.

So if you did `ord wallet --name test1 create` a new directory would be created in `umbrel/app-data/bitcoin/data/bitcoin` called `test` with the wallet.dat file in it.

I just woke up though, so maybe I'm misunderstanding this code and will look more at it in a bit

Reply:
It's all good, this is for the ord wallet database files (which are redb) not the bitcoind wallet.dat files.

I tested with multiple wallet database files on different networks and the migration works well.

Reply:
Okay, I see now.  Yeah, looks good reading it over again.  I haven't looked into how I can test an app without being in the app store or a community app store, but I think the code looks good